src/components/NavigationBar/NavigationLinksItem.vue

```diff
@@ -1,40 +1,35 @@
+<script lang="ts" setup>
+import { computed, toRef } from 'vue'
+import { useRoute } from 'vue-router'
+import Icon from '/@/components/UI/Icon.vue'
+import useRouteInfo from '/@/use/routeInfo'
+
+interface Props {
+  name: string
+  path: string
+}
+
+const currentRoute = useRoute()
+
+const props = defineProps<Props>()
+
+const routeInfo = useRouteInfo(toRef(props, 'name'))
+
+const isActive = computed(() => {
+  if (props.path === '/') return currentRoute.path === props.path
+  return currentRoute.path.startsWith(`${props.path}`)
```

Comment:
変数と文字列混在してるわけではないのでこれでよさそうです？
```suggestion
  return currentRoute.path.startsWith(props.path)
```